frontend/src/views/WheresMyGene/components/Filters/index.tsx

```diff
@@ -322,10 +326,28 @@ export default memo(function Filters({
     [handleFilterChange]
   );
 
-  const handleTissuesChange = useMemo(
-    () => handleFilterChange("tissues"),
-    [handleFilterChange]
-  );
+  const handleTissuesChange = useMemo(() => {
+    handleFilterChange("tissues");
```

Comment:
I think `handleFilterChange("tissues")` returns a callback function (instead of triggering a side effect) that was invoked with `onChange={handleTissuesChange}`, but now it's not being used at all, since the returned function is never invoked. Maybe that's why you had to create a new slice of state?

CC: @seve to confirm my hypothesis